meilisearch-schema/Cargo.toml

```diff
@@ -11,3 +11,4 @@ indexmap = { version = "1.3.2", features = ["serde-1"] }
 serde = { version = "1.0.105", features = ["derive"] }
 serde_json = { version = "1.0.50", features = ["preserve_order"] }
 toml = { version = "0.5.6", features = ["preserve_order"] }
+zerocopy = "0.3.0"
```

Comment:
Why have you added this dependency? Is it related to the `attributes_for_faceting` that is an array and not an `HashSet` now?